unit_test/test_spdm_requester/get_event_types.c

```diff
@@ -214,8 +214,14 @@ static libspdm_return_t receive_message(
 
         generate_dmtf_event_group(spdm_response + 1, &event_group_total_bytes,
                                   true, true, true, true);
-        spdm_response->supported_event_groups_list_len = event_group_total_bytes;
 
+        {
+            /* Workaround. See issue https://github.com/DMTF/libspdm/issues/2645. The below code
+             * can be removed when the issue has been resolved. */
+            event_group_total_bytes += 2;
```

Comment:
This workaround can be removed, after https://github.com/DMTF/libspdm/pull/2650 is merged.

Reply:
@steven-bellock , 2650 is merged. Could you please remove this work-around and try again?